_shared_assets/themes/owncloud_com/layout.html

```diff
@@ -96,6 +96,22 @@
   });
 }());
 </script>
+
+<!-- Piwik -->
+<script type="text/javascript">
+  var _paq = _paq || [];
+  /* tracker methods like "setCustomDimension" should be called before "trackPageView" */
+  _paq.push(['trackPageView']);
+  _paq.push(['enableLinkTracking']);
+  (function() {
+    var u="//piwik.owncloud.com/";
```

Comment:
You can directly write `var u="https://piwik.owncloud.com/";` as `http://` anyway gets redirected to `https://`

Reply:
Fair enough. I just took the supplied code.